go/vt/mysqlctl/cephbackupstorage/ceph.go

```diff
@@ -88,7 +88,11 @@ func (bh *CephBackupHandle) AddFile(ctx context.Context, filename string, filesi
 
 		// Give PutObject() the read end of the pipe.
 		object := objName(bh.dir, bh.name, filename)
-		_, err := bh.client.PutObjectWithContext(ctx, bucket, object, reader, -1, minio.PutObjectOptions{ContentType: "application/octet-stream"})
+		// if filesize is given as 0, pass it as -1 = UNKNOWN
+		if filesize == 0 {
```

Comment:
I think this check is not required. -1 was given when file size was not known i.e. not passed to this method. But, now the file size is known this will work irrespective if file size is actually 0 Bytes.

Reply:
The calling code passes 0 when the file size is actually unknown. This happens when we write the manifest file. We call `AddFile` to get a `WriteCloser` and we actually write to the file using that.
I can add this as a comment so that it is clearer to anyone reading the code.

Reply:
I like the other option of passing -1 for unknown file size so no special check is required in this code block and would be clean.